src/components/tree-view/TreeView.tsx

```diff
@@ -159,14 +166,18 @@ export const TreeView = <T,>({
           if (canDrop) {
             const canDropResult = canDrop({ parentNode, dragNodes, index });
             if (!canDropResult) {
-              parentNode.data.value.canDrop = false;
+              if (parentNode.id !== "__REACT_ARBORIST_INTERNAL_ROOT__") {
```

Comment:
Can you add a comment explaining this part of the code, it isn't quite easy to understand coming to the project imo